java.yaml

```diff
@@ -11,9 +11,13 @@ steps:
   - '*|*=gcr.io/google-appengine/jetty:9'
   - 'openjdk8|*=gcr.io/google-appengine/jetty:9'
   - 'openjdk8|jetty9=gcr.io/google-appengine/jetty:9'
+  - 'openjdk8|jetty=gcr.io/google-appengine/jetty:9'
```

Comment:
I see your reasoning now. If we never support more than one version of a server per version of JDK, it makes perfect sense to not use :latest when JDK is specified, but to use it when JDK is not specified.